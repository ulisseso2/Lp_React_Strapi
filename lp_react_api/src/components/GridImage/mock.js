export default {
  title: 'Grid de Imagens',
  description: `Lorem ipsum dolor sit, amet consectetur adipisicing elit. Adipisci numquam explicabo error omnis cumque.`,
  image: [
    {
      srcImg: 'https://source.unsplash.com/random/800x800?r=1',
      alt: 'Foto 1',
    },
    {
      srcImg: 'https://source.unsplash.com/random/800x800?r=2',
      alt: 'Foto 2',
    },
    {
      srcImg: 'https://source.unsplash.com/random/800x800?r=3',
      alt: 'Foto 3',
    },
    {
      srcImg: 'https://source.unsplash.com/random/800x800?r=4',
      alt: 'Foto 3',
    },
    {
      srcImg: 'https://source.unsplash.com/random/800x800?r=5',
      alt: 'Foto 3',
    },
    {
      srcImg: 'https://source.unsplash.com/random/800x800?r=6',
      alt: 'Foto 3',
    },
  ],
};
