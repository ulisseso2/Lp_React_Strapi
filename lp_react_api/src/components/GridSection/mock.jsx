export default {
  background: false,
  title: 'My grid',
  description: `Assumenda, animi praesentium voluptas velit voluptatum quam repudiandae, impedit minus consequatur iusto atque voluptatem numquam, aliquam tempore ratione culpa facere asperiores inventore?`,
  grid: [
    {
      title: 'Teste a',
      description: `Lorem 1 ipsum dolor sit amet consectetur adipisicing elit. Architecto rem cupiditate dolorum voluptatem in pariatur cumque exercitationem molestias aliquid. Magni veniam numquam, reiciendis voluptas aliquid ut saepe vero vel ratione!`,
    },
    {
      title: 'Teste b',
      description: `Lorem 2 ipsum dolor sit amet consectetur adipisicing elit. Architecto rem cupiditate dolorum voluptatem in pariatur cumque exercitationem molestias aliquid. Magni veniam numquam, reiciendis voluptas aliquid ut saepe vero vel ratione!`,
    },
    {
      title: 'Teste c',
      description: `Lorem 3 ipsum dolor sit amet consectetur adipisicing elit. Architecto rem cupiditate dolorum voluptatem in pariatur cumque exercitationem molestias aliquid. Magni veniam numquam, reiciendis voluptas aliquid ut saepe vero vel ratione!`,
    },
  ],
};
