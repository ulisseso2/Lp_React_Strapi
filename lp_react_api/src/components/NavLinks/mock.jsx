export default [
  {
    children: 'link 1',
    link: '#target1',
    newTab: false,
  },
  {
    children: 'link 2',
    link: '#target2',
    newTab: false,
  },
  {
    children: 'link 3',
    link: '#target3',
    newTab: false,
  },
  {
    children: 'link 4',
    link: '#target4',
    newTab: false,
  },
  {
    children: 'link 5',
    link: '#target5',
    newTab: false,
  },
];
