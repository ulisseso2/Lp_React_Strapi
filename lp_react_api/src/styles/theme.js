export const theme = {
  colors: {
    primary: '#457B9D',
    light: '#A8DADC',
    dark: '#1D3557',
    hot: '#E63946',
    white: '#f7fff7',
    black: '#343434',
  },
  fonts: {
    family: {
      main: "'Open Sans', sans-serif",
      title: "'Montserrat', sans-serif",
      body: "'Open Sans', sans-serif",
    },
  },
};
//tema padrão da aplicação usado em vários componentes
