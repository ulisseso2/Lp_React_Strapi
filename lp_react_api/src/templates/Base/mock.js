export default {
  links: [
    {
      children: 'link 1',
      link: '#target1',
      newTab: false,
    },
    {
      children: 'link 2',
      link: '#target2',
      newTab: false,
    },
    {
      children: 'link 3',
      link: '#target3',
      newTab: false,
    },
    {
      children: 'link 4',
      link: '#target4',
      newTab: false,
    },
    {
      children: 'link 5',
      link: '#target5',
      newTab: false,
    },
  ],
  logoData: {
    text: 'Algo',
    srcImg: 'assets/images/logo_cor_horiz.png',
    link: 'http://localhost.com',
  },
  footerHtml: '<p>Teste de footer</p>',
};
