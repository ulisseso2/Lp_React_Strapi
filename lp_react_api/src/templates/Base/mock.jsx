import { GridSection } from '../../components/GridSection';
import mockGrid from '../../components/GridSection/mock';
import mockGridImage from '../../components/GridImage/mock';
import { GridImage } from '../../components/GridImage';
import mockNavLink from '../../components/NavLinks/mock';
import { GridTwoColumns } from '../../components/GridTwoColumn';
import mockTwoColumns from '../../components/GridTwoColumn/mock';

export const mockBase = {
  children: (
    <>
      <GridSection {...mockGrid} background />
      <GridTwoColumns {...mockTwoColumns} />
      <GridSection {...mockGrid} background />
      <GridSection {...mockGrid} />
      <GridImage {...mockGridImage} />
    </>
  ),
  links: mockNavLink,
  logoData: {
    text: 'Algo',
    srcImg: 'assets/images/logo_cor_horiz.png',
    link: 'http://localhost.com',
  },
  footerHtml:
    '<p> <a href="https://linkedin.com/in/ulissesoliveira2" target="_blank">Feito com <span>‚ù§</span> por Ulisses Oliveira </a></p>',
};
