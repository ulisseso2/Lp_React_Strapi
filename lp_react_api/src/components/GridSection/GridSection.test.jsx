import { screen } from '@testing-library/react';
import { GridSection } from '.';
import { renderTheme } from '../../styles/render.theme';

import mock from './mock';

describe('<GridSection>', () => {
  it('should render with default values', () => {
    const { container } = renderTheme(<GridSection {...mock} />);
    expect(container).toMatchSnapshot();
  });
  it('should render with default values', () => {
    const { container } = renderTheme(
      <GridSection {...mock} background={undefined} />,
    );
    expect(container).toMatchSnapshot();
  });
});
