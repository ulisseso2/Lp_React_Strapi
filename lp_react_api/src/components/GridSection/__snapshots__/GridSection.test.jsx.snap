// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GridSection> should render with default values 1`] = `
<div>
  <div
    class="sc-gKXOVf dgAgri"
    id=""
  >
    <div
      class="sc-jSMfEi cRkChI"
    >
      <div
        class="sc-dkzDqf budGeo"
      >
        <h2
          class="sc-bczRLJ hGDuJC"
        >
          My grid
        </h2>
        <div
          class="sc-gsnTZi cUyYvE"
        >
          Assumenda, animi praesentium voluptas velit voluptatum quam repudiandae, impedit minus consequatur iusto atque voluptatem numquam, aliquam tempore ratione culpa facere asperiores inventore?
        </div>
        <div
          class="sc-hKMtZM gRfbNB"
        >
          <div
            class="sc-eCYdqJ jWNvvV"
          >
            <h3
              class="sc-bczRLJ fnHBwd"
            >
              Teste a
            </h3>
            <div
              class="sc-gsnTZi cUyYvE"
            >
              Lorem 1 ipsum dolor sit amet consectetur adipisicing elit. Architecto rem cupiditate dolorum voluptatem in pariatur cumque exercitationem molestias aliquid. Magni veniam numquam, reiciendis voluptas aliquid ut saepe vero vel ratione!
            </div>
          </div>
          <div
            class="sc-eCYdqJ jWNvvV"
          >
            <h3
              class="sc-bczRLJ fnHBwd"
            >
              Teste b
            </h3>
            <div
              class="sc-gsnTZi cUyYvE"
            >
              Lorem 2 ipsum dolor sit amet consectetur adipisicing elit. Architecto rem cupiditate dolorum voluptatem in pariatur cumque exercitationem molestias aliquid. Magni veniam numquam, reiciendis voluptas aliquid ut saepe vero vel ratione!
            </div>
          </div>
          <div
            class="sc-eCYdqJ jWNvvV"
          >
            <h3
              class="sc-bczRLJ fnHBwd"
            >
              Teste c
            </h3>
            <div
              class="sc-gsnTZi cUyYvE"
            >
              Lorem 3 ipsum dolor sit amet consectetur adipisicing elit. Architecto rem cupiditate dolorum voluptatem in pariatur cumque exercitationem molestias aliquid. Magni veniam numquam, reiciendis voluptas aliquid ut saepe vero vel ratione!
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
   
</div>
`;

exports[`<GridSection> should render with default values 2`] = `
<div>
  <div
    class="sc-gKXOVf dgAgri"
    id=""
  >
    <div
      class="sc-jSMfEi cRkChI"
    >
      <div
        class="sc-dkzDqf budGeo"
      >
        <h2
          class="sc-bczRLJ hGDuJC"
        >
          My grid
        </h2>
        <div
          class="sc-gsnTZi cUyYvE"
        >
          Assumenda, animi praesentium voluptas velit voluptatum quam repudiandae, impedit minus consequatur iusto atque voluptatem numquam, aliquam tempore ratione culpa facere asperiores inventore?
        </div>
        <div
          class="sc-hKMtZM gRfbNB"
        >
          <div
            class="sc-eCYdqJ jWNvvV"
          >
            <h3
              class="sc-bczRLJ fnHBwd"
            >
              Teste a
            </h3>
            <div
              class="sc-gsnTZi cUyYvE"
            >
              Lorem 1 ipsum dolor sit amet consectetur adipisicing elit. Architecto rem cupiditate dolorum voluptatem in pariatur cumque exercitationem molestias aliquid. Magni veniam numquam, reiciendis voluptas aliquid ut saepe vero vel ratione!
            </div>
          </div>
          <div
            class="sc-eCYdqJ jWNvvV"
          >
            <h3
              class="sc-bczRLJ fnHBwd"
            >
              Teste b
            </h3>
            <div
              class="sc-gsnTZi cUyYvE"
            >
              Lorem 2 ipsum dolor sit amet consectetur adipisicing elit. Architecto rem cupiditate dolorum voluptatem in pariatur cumque exercitationem molestias aliquid. Magni veniam numquam, reiciendis voluptas aliquid ut saepe vero vel ratione!
            </div>
          </div>
          <div
            class="sc-eCYdqJ jWNvvV"
          >
            <h3
              class="sc-bczRLJ fnHBwd"
            >
              Teste c
            </h3>
            <div
              class="sc-gsnTZi cUyYvE"
            >
              Lorem 3 ipsum dolor sit amet consectetur adipisicing elit. Architecto rem cupiditate dolorum voluptatem in pariatur cumque exercitationem molestias aliquid. Magni veniam numquam, reiciendis voluptas aliquid ut saepe vero vel ratione!
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
   
</div>
`;
